BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.h

```diff
@@ -192,7 +192,7 @@
  *                          as rootCacheDir for main app to allow main app takes over background session
  *                          tasks created from extensions
  */
-+ (void)oneTimeSetUpInExtensionToSupportBackgroundTasksWithDelegate:(id<BOXURLSessionManagerDelegate>)delegate rootCacheDir:(nonnull NSString *)rootCacheDir;
++ (void)oneTimeSetUpInExtensionToSupportBackgroundTasksWithDelegate:(id<BOXURLSessionManagerDelegate>)delegate rootCacheDir:(NSString *)rootCacheDir;
```

Comment:
using nonnull in here makes it a requirement to specify nullable or not for every property in the class